### PR TITLE
Add section  on section version management

### PR DESCRIPTION
**Pull request by danielhuppmann**

### CODE REVIEW

File: scenario-databases.rst [hunk 1]
```diff
@@ -57,7 +56,15 @@ are required.
...
 
-Scenario submission
--------------------
+Scenario version management
+---------------------------
 
-Coming soon
\ No newline at end of file
+When submitting a scenario (a.k.a. "run") to an IIASA database instance with an already
+existing model-scenario name, the database will save the new submission as a new version
```
Comment (phackstock) [inline scenario-databases.rst]: Maybe use the word _combination_ for emphasis:
```suggestion
existing model-scenario combination, the database will save the new submission as a new version
```

File: scenario-databases.rst [hunk 2]
```diff
@@ -57,7 +56,15 @@ are required.
...
+of that run. The **version number** is assigned automatically and the new version
```
Comment (phackstock) [inline scenario-databases.rst]:
```suggestion
of that run. The **version number** is incremented automatically and the new version
```

**Review by phackstock — APPROVED** [review]
Good to merge from my side.
Two small comments in line below but good to merge either way.

**Comment by danielhuppmann** [comment]
Thanks @phackstock

**Pull request merged**